### PR TITLE
feat(ui): Add session status release comparison charts

### PR DESCRIPTION
**Pull request by matejminar**
Adding 8 more charts to release comparison:
- Healthy Session+User Rate
- Abnormal Session+User Rate
- Errored Session+User Rate
- Crashed Session+User Rate

image

This is still feature flagged, leaving styling for the next PR, this one is already quite long.

### CODE REVIEW

File: static/app/views/releases/detail/overview/releaseComparisonChart/index.tsx [hunk 1]
```diff
@@ -201,6 +207,66 @@ function ReleaseComparisonChart({
       ? releaseCrashFreeSessions - allCrashFreeSessions
       : null;
 
+  const releaseHealthySessions = getSessionStatusRate(
```
Comment (priscilawebdev) [inline static/app/views/releases/detail/overview/releaseComparisonChart/index.tsx:210]: oh wow this file is very long 😮

File: static/app/views/releases/utils/index.tsx [hunk 2]
```diff
@@ -48,6 +48,14 @@ export const displayCrashFreePercent = (
   return `${rounded}\u0025`;
 };
 
+export const getSessionStatusPercent = (percent: number): number => {
+  return round(percent, 3);
+};
+
+export const displaySessionStatusPercent = (percent: number): string => {
```
Comment (priscilawebdev) [inline static/app/views/releases/utils/index.tsx:55]:
```suggestion
export const displaySessionStatusPercent = (percent: number) => {
```

File: static/app/views/releases/utils/index.tsx [hunk 3]
```diff
@@ -48,6 +48,14 @@ export const displayCrashFreePercent = (
   return `${rounded}\u0025`;
 };
 
+export const getSessionStatusPercent = (percent: number): number => {
```
Comment (priscilawebdev) [inline static/app/views/releases/utils/index.tsx:51]:
```suggestion
export const getSessionStatusPercent = (percent: number) => {
```

**Review by priscilawebdev — APPROVED** [review]
code looks good to me

File: static/app/views/releases/detail/overview/releaseComparisonChart/index.tsx (hunk 1 above)
Comment (matejminar) [reply static/app/views/releases/detail/overview/releaseComparisonChart/index.tsx:210]: Right now it is. I'll be adding conditions to what charts you have access (performance, discover, sessions) and that's when I plan to break it down 👍

**Pull request merged**